projects/laji/test/+user/user.spec.ts

```diff
@@ -20,20 +11,17 @@ test.describe('User page', () => {
   });
 
   test.afterEach(async () => {
-    expect(await page.locator('.toast-error').isVisible(), 'Error dialog is present.').toBe(false);
+    await expect(page.locator(ERROR_DIALOG_SELECTOR)).not.toBeVisible();
   });
 
   test('should login user', async () => {
     await login(page);
-
-    const usernameElem = page.locator('#logged-in-user');
-    expect(await usernameElem.isVisible()).toBe(true);
+    await expect(page.locator('#logged-in-user')).toBeVisible();
   });
 
   test('should logout user', async () => {
     await logout(page);
-
     await page.locator('#logged-in-user').waitFor({ state: 'detached' });
```

Comment:
Btw I don't think we a need for the `waitFor` usage in the test cases. Seems redundant since there's the visibility assertion below this line already. In some cases this can fix flakiness but you can achieve the same thing by just increasing the timeout of the assertion.